src/txpool/getters.yaml

```diff
@@ -0,0 +1,69 @@
+- name: txpool_transactions
+  summary: Returns a list of all transactions that match the supplied filter conditions that are either pending for inclusion in the next block(s) or scheduled for future execution.
+  params:
+    - name: Filter
+      schema:
+        title: txpool filter
```

Comment:
Having now different kind of txs, probably a `type` filter could be useful

Reply:
Do you mean 4844/blobs transactions? if so, also see other comment that from my understanding 4844/blobs will have its own transaction pool and will not be included for this proposal.

Reply:
I mean that the filter could accept a `type` field where you can specify any of the transaction types: Frontier, EIP2930, EIP1559, EIP4844